docker/postgres/docker-entrypoint-initdb.d/mock-data/interfaith_DDL.sql

```diff
@@ -54,22 +50,25 @@ CREATE TABLE needs (
   need_active BOOLEAN NOT NULL
 );
 
-DROP TABLE IF EXISTS needcallouts cascade;
-CREATE TABLE needcallouts (
-  needcallout_id BIGSERIAL PRIMARY KEY,
-  needcallout_quantity INT NOT NULL,
+CREATE TABLE calloutneeds (
+  calloutneed_id BIGSERIAL PRIMARY KEY,
+  calloutneed_quantity INT NOT NULL,
   need_id BIGINT REFERENCES needs,
   callout_id BIGINT REFERENCES callouts,
-  needcallout_active BOOLEAN NOT NULL
+  calloutneed_active BOOLEAN NOT NULL
 );
 
-DROP TABLE IF EXISTS donations cascade;
 CREATE TABLE donations (
   donation_id BIGSERIAL PRIMARY KEY,
   donation_quantity INT NOT NULL,
   donation_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
-  needcallout_id BIGINT REFERENCES users NOT NULL,
+  calloutneed_id BIGINT REFERENCES users NOT NULL,
   -- donor_id BIGINT REFERENCES donors NOT NULL
 );
 
--- transaction stuff, activations, 
+CREATE TABLE alerts (
+  alert_id BIGSERIAL PRIMARY KEY,
+  alert_sent_date TIMESTAMP NULL,
```

Comment:
I'm assuming sent_date is the published date? Should it be nullable?

Reply:
Yep. My assumption was that, let's say someone creates an alert, but doesn't want to send it out immediately. They can send it out at a later time by sending it. We can clarify this at the next meeting if this is what an alert should do.

Reply:
I agree, though it seems like a question of if Jody wants to have that functionality or if she just wants to push an alert when she presses a button and doesn't care about scheduling future ones. I could really see her going either way on it